### PR TITLE
Report button too close to reply button

### PR DESCRIPTION
**Pull request by srbaker**
In the comments feed, the report button was placed next to the reply button.  This ran the risk of accidentally reporting when a reply was intended.

This change moves the Report action under the context menu, as it is a less common action and this will prevent accidental reports.

To verify this, report something by clicking Report under the three-dots context menu.

### CODE REVIEW

**Comment by hennevogel** [comment]
https://github.com/openSUSE/open-build-service/blob/master/CONTRIBUTING.md#proper-pull-request

File: src/api/app/components/bs_request_comment_component.html.haml [hunk 1]
```diff
@@ -24,6 +24,15 @@
                                     action: comment_path(comment) },
                             class: 'dropdown-item delete_link', title: 'Delete comment') do
                 Delete
+            - if Flipper.enabled?(:content_moderation, User.session) and policy(Report.new(reportable: comment)).create?
```
Comment (krauselukas) [inline src/api/app/components/bs_request_comment_component.html.haml:27]:
```suggestion
            - if policy(Report.new(reportable: comment)).create?
```

**Review by krauselukas — COMMENTED** [review]
@srbaker could you please get rid of the merge/update commits

File: src/api/app/components/bs_request_comment_component.html.haml [hunk 2]
```diff
@@ -24,6 +24,15 @@
...
+              = link_to('#', class: 'dropdown-item', id: "js-comment-#{comment.id}",
+                        data: { 'bs-toggle': 'modal',
+                                'bs-target': '#report-modal',
+                                'modal-title': "Report comment from #{comment.user}",
+                                'reportable-type': comment.class.name,
+                                'reportable-id': comment.id }) do
+                Report
```
Comment (hellcp-work) [inline src/api/app/components/bs_request_comment_component.html.haml:34]: Can you move the Report button above Delete?

**Comment by hennevogel** [comment]
[Report button is too close to the reply button (S)](https://trello.com/c/L3Gb6fkd/2555-report-button-is-too-close-to-the-reply-button-s)

**Review by krauselukas — APPROVED** [review]

**Pull request merged**